students/wachulec_maria/lesson_10_organizing_files/deleting_unneeded_files.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+
+def search_large_files(folder_tree):
+    for folders, subfolders, files in os.walk(folder_tree):
+        for i in files:
+            if os.path.getsize(os.path.join(folders, i)) > 100000000:
```

Comment:
would be much better to export values like 100000000 into separate var, preferably CONST